_config.yml

```diff
@@ -90,6 +90,7 @@ index_old_doc_paths: false
 # drop-down list in the distro selector.
 #
 ros2_distros:
+  - 'iron'
```

Comment:
Can you keep this list in alphabetical order. It looks like humble was inserted incorrectly and should be changed too.

Reply:
Gotcha [097620b](https://github.com/ros-infrastructure/rosindex/pull/306/commits/097620bbd31ffdf78aee030fa6d6616e0cb81482)